### PR TITLE
style: align Heading and Items

### PR DESCRIPTION
**Pull request by olegfedak**
**Summary**
It's just a small fix of styles without changing the design.

**As a result**
decap-style-fix

### CODE REVIEW

**Review by martinjagodic — APPROVED** [review]

**Review by martinjagodic — APPROVED** [review]

**Review by olegfedak — COMMENTED** [review]
**Summary**
It's just a small fix of styles without changing the design.

**As a result**
Знімок екрана 2023-11-24 о 14 55 14

**Review by martinjagodic — APPROVED** [review]
@olegfedak this looks nice, thanks! If you indend to add more changes to this PR please convert it to draft. It doesn't make much sense to review every commit :)

**Comment by olegfedak** [comment]
> @olegfedak this looks nice, thanks! If you indend to add more changes to this PR please convert it to draft. It doesn't make much sense to review every commit :)

Oh, I will! It seems I didn't completely understand how the pull request works. I might should work in a branch from the forked repo master

**Comment by martinjagodic** [comment]
That's ok. Mark it as ready for review when you're done.

**Review by olegfedak — COMMENTED** [review]
**Result:**
Знімок екрана 2023-12-02 о 00 34 06

**Review by martinjagodic — APPROVED** [review]

File: dev-test/config.yml
Comment (martinjagodic) [inline dev-test/config.yml:1]: Removing restaurants and color is not in the scope of this PR. Please add it back.

**Review by martinjagodic — CHANGES_REQUESTED** [review]

**Review by martinjagodic — APPROVED** [review]

**Pull request merged**